<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notes</class>
 <widget class="QWidget" name="Notes">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial Narrow</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Notes</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/icon.png</normaloff>icons/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 242, 165)</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btn1">
      <property name="toolTip">
       <string>create</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(177, 255, 149)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/add-button.png</normaloff>icons/add-button.png</iconset>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btn3">
      <property name="toolTip">
       <string>delete</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 97, 97)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/icon-recycle-bin.png</normaloff>icons/icon-recycle-bin.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn2">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>edit</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(171, 172, 163)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/edit.png</normaloff>icons/edit.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>241</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 245, 124)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="overwriteMode">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 245, 124)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
